### PR TITLE
Simple proof of concept for file readers/ writers

### PR DESCRIPTION
**Pull request by toon-leader-bacon**
This is a merge request for adding the ability to read and write files to our Julia project.

### CODE REVIEW

**Comment by lucasmccabe** [comment]
I like the functionality you're adding, but can we better integrate it with the package structure? I'm wary of merging a parallel `nocab` submodule, etc. How about something like `TextUtils.jl` under `src/juisee`?

File: .gitignore [hunk 1]
```diff
@@ -22,3 +22,7 @@ docs/site/
 # committed for packages, but should be committed for applications that require a static
 # environment.
 Manifest.toml
+
+tmp/*
+!tmp/README.md
+nocab/*
```
Comment (toon-leader-bacon) [inline .gitignore:28]: My personal playspace. Don't want to pollute your guys' development space with my experiements 🧑‍🔬

File: src/juisee/TextUtils.jl [hunk 2]
```diff
@@ -1,12 +1,117 @@
-# TextUtils.jl
```
Comment (toon-leader-bacon) [inline src/juisee/TextUtils.jl:1]: I'll add this part back in

**Comment by lucasmccabe** [comment]
LGTM - thanks for working on this!

**Pull request merged**